diff -ruN mysqlBind.orig/bind.c mysqlBind/bind.c
--- mysqlBind.orig/bind.c	2003-12-22 17:27:39.000000000 +0100
+++ mysqlBind/bind.c	2004-03-01 13:54:00.000000000 +0100
@@ -22,6 +22,7 @@
 */
 
 #include "mysqlrad.h"
+#include "local.h"
 
 
 void PassDirectHtml(char *file);
@@ -545,15 +546,15 @@
 	Header_ism3("Import()",0);
 	printf("</center><font size=2><blockquote>\n");
 
-	printf("Importing master zones from /usr/local/mysqlbind/import directory. File name must be same as zone name. Only A, CNAME, MX, PTR and NS records supported. SOA ttl values must be seconds. SOA NS must be in tNameServer.\n");
+	printf("Importing master zones from " NAMED_CONF "/import directory. File name must be same as zone name. Only A, CNAME, MX, PTR and NS records supported. SOA ttl values must be seconds. SOA NS must be in tNameServer.\n");
 
 	printf("<pre>\n");
 		
-	n=scandir("/usr/local/mysqlbind/import", &namelist, 0, 0);
+	n=scandir(NAMED_CONF "/import", &namelist, 0, 0);
 			
 	if(n<0)
 	{
-		printf("scandir() error: Does /usr/local/mysqlbind/import exist?\n");
+		printf("scandir() error: Does " NAMED_CONF "/import exist?\n");
 		Footer_ism3();
 	}
 	else if(n==2)
@@ -574,7 +575,7 @@
 	{
 		FILE *fp;
 
-		sprintf(query,"/usr/local/mysqlbind/import/%.100s", namelist[i]->d_name);
+		sprintf(query,NAMED_CONF "/import/%.100s", namelist[i]->d_name);
 		if(!(fp=fopen(query,"r")))
 		{
 			printf("Could not open: %s\n",query);
@@ -927,7 +928,7 @@
 			strcpy(query,"a");
 		else
 			strcpy(query,"w");
-		if(!(sfp=fopen("/usr/local/mysqlbind/named.d/master.zones",query)))
+		if(!(sfp=fopen(NAMED_CONF "/named.d/master.zones",query)))
 		{
 			fprintf(stderr,"Could not open master.zones\n");
 			exit(1);
@@ -955,7 +956,7 @@
 		char *cp;
 
 		//write master db file
-		sprintf(query,"/usr/local/mysqlbind/named.d/master/%c/%s",field[0][0],field[0]);
+		sprintf(query,NAMED_CONF "/named.d/master/%c/%s",field[0][0],field[0]);
 		if(!(zfp=fopen(query,"w")))
 		{
 			fprintf(stderr,"Could not open master zone db file\n");
@@ -1076,7 +1077,7 @@
 		strcpy(query,"a");
 	else
 		strcpy(query,"w");
-	if(!(fp=fopen("/usr/local/mysqlbind/named.d/slave.zones",query)))
+	if(!(fp=fopen(NAMED_CONF "/named.d/slave.zones",query)))
 	{
 		fprintf(stderr,"Could not open slave.zones\n");
 		exit(1);
@@ -1124,50 +1125,29 @@
 	printf("Installing named for IP %s from %s/mysqlBind\n",
 			IPNumber(cIpNum), cISMROOT);
 	
-	mkdir("/usr/local/mysqlbind",0777);
-	mkdir("/usr/local/mysqlbind/named.d",0777);
-	mkdir("/usr/local/mysqlbind/named.d/master",0777);
-	mkdir("/usr/local/mysqlbind/named.d/slave",0777);
-
 	printf("Configuring and installing files...\n");
 
-	//Create dir hierarchy	
-	for(i='0';i<='9';i++)
-	{
-		sprintf(query,"/usr/local/mysqlbind/named.d/master/%c",i);
-		mkdir(query,0777);
-		sprintf(query,"/usr/local/mysqlbind/named.d/slave/%c",i);
-		mkdir(query,0777);
-	}
-	for(i='a';i<='z';i++)
-	{
-		sprintf(query,"/usr/local/mysqlbind/named.d/master/%c",i);
-		mkdir(query,0777);
-		sprintf(query,"/usr/local/mysqlbind/named.d/slave/%c",i);
-		mkdir(query,0777);
-	}
-
-	sprintf(query,"cat %s/mysqlBind/%s/named.conf|sed -e \"s/{{cIpNumber}}/%s/g\" > /usr/local/mysqlbind/named.conf",cISMROOT,cSetupDir,IPNumber(cIpNum));	
+	sprintf(query,"cat %s/mysqlBind/%s/named.conf|sed -e \"s/{{cIpNumber}}/%s/g\" > " NAMED_CONF "/named.conf",cISMROOT,cSetupDir,IPNumber(cIpNum));	
 	if(system(query))
 		 printf("Error configuring named.conf\n");
 
-	sprintf(query,"touch /usr/local/mysqlbind/named.d/master.zones");	
+	sprintf(query,"touch " NAMED_CONF "/named.d/master.zones");	
 	if(system(query))
 		 printf("Error configuring master.zones\n");
 
-	sprintf(query,"touch /usr/local/mysqlbind/named.d/slave.zones");	
+	sprintf(query,"touch " NAMED_CONF "/named.d/slave.zones");	
 	if(system(query))
 		 printf("Error configuring slave.zones\n");
 
-	sprintf(query,"cat %s/mysqlBind/%s/localhost > /usr/local/mysqlbind/named.d/master/localhost",cISMROOT,cSetupDir);	
+	sprintf(query,"cat %s/mysqlBind/%s/localhost > " NAMED_CONF "/named.d/master/localhost",cISMROOT,cSetupDir);	
 	if(system(query))
 		 printf("Error configuring localhost\n");
 
-	sprintf(query,"cat %s/mysqlBind/%s/127.0.0 > /usr/local/mysqlbind/named.d/master/127.0.0",cISMROOT,cSetupDir);	
+	sprintf(query,"cat %s/mysqlBind/%s/127.0.0 > " NAMED_CONF "/named.d/master/127.0.0",cISMROOT,cSetupDir);	
 	if(system(query))
 		 printf("Error configuring 127.0.0\n");
 
-	sprintf(query,"cat %s/mysqlBind/%s/root.cache > /usr/local/mysqlbind/named.d/root.cache",cISMROOT,cSetupDir);	
+	sprintf(query,"cat %s/mysqlBind/%s/root.cache > " NAMED_CONF "/named.d/root.cache",cISMROOT,cSetupDir);	
 	if(system(query))
 		 printf("Error configuring root.cache\n");
 
@@ -1557,7 +1537,7 @@
 		//debug only
 		printf("Modify\n\n");
 		//Single zone, DBFiles yes, Stubs no
-		sprintf(query,"grep 'zone \"%s\"' /usr/local/mysqlbind/named.d/master.zones > /dev/null",cZone);
+		sprintf(query,"grep 'zone \"%s\"' " NAMED_CONF "/named.d/master.zones > /dev/null",cZone);
 		if(system(query))
 		{
 			CreateMasterFiles(cMasterNS,"",1,1);
--- ./mainfunc.h.org	2006-09-29 08:42:31.000000000 +0000
+++ ./mainfunc.h	2006-09-29 08:50:09.000000000 +0000
@@ -13,6 +13,8 @@
 	ConnectDb?
 */
 
+#include "local.h"
+
 
 static char cTableList[32][32]=\
 {"tZone","tResource","tRRType","tJob","tNameServer","tMailServer","tClient","tAuthorize","tConfiguration",""};
@@ -320,7 +322,7 @@
 {
 	Header_ism3("NamedConf()",0);
 	printf("</center><pre><blockquote>");
-	PassDirectHtml("/usr/local/mysqlbind/named.conf");
+	PassDirectHtml(NAMED_CONF "/named.conf");
 	printf("</blockquote></pre>");
 	Footer_ism3();
 }//void NamedConf(void)
@@ -361,12 +363,8 @@
 		strncpy(cISMROOT,getenv("ISMROOT"),255);
 		cHost[255]=0;
 	}
-
-	if(!cISMROOT[0])
-	{
-		printf("You must set ISMROOT env var first. Ex. (bash) export ISMROOT=/home/ism-3.0\n");
-	exit(1);
-	}
+	else
+		sprintf(cISMROOT,LIBDIR);
 
 	printf("Restoring mysqlBind data from .txt file in %s/{{cProject}}/data...\n\n",cISMROOT);
 
@@ -407,12 +405,8 @@
 		strncpy(cISMROOT,getenv("ISMROOT"),255);
 		cHost[255]=0;
 	}
-
-	if(!cISMROOT[0])
-	{
-		printf("You must set ISMROOT env var first. Ex. (bash) export ISMROOT=/home/ism-3.0\n");
-	exit(1);
-	}
+	else
+		sprintf(cISMROOT,LIBDIR);
 
 	printf("Restoring mysqlBind data from .txt file in %s/{{cProject}}/data...\n\n",cISMROOT);
 
@@ -449,12 +443,8 @@
 		strncpy(cISMROOT,getenv("ISMROOT"),255);
 		cHost[255]=0;
 	}
-
-	if(!cISMROOT[0])
-	{
-		printf("You must set ISMROOT env var first. Ex. (bash) export ISMROOT=/home/ism-3.0\n");
-	exit(1);
-	}
+	else
+		sprintf(cISMROOT,LIBDIR);
 
 	printf("Backing up mysqlBind data to .txt files in \
 %s/mysqlBind/data...\n\n",cISMROOT);
@@ -505,24 +495,8 @@
 		strncpy(cISMROOT,getenv("ISMROOT"),255);
 		cISMROOT[255]=0;
 	}
-
-	if(!cISMROOT[0])
-	{
-		printf("You must set ISMROOT env var first. Ex. (bash) export ISMROOT=/home/joe\n");
-		exit(1);
-	}
-
-	if(getenv("CGIDIR")!=NULL)
-	{
-		strncpy(cCGIDIR,getenv("CGIDIR"),255);
-		cCGIDIR[255]=0;
-	}
-
-	if(!cCGIDIR[0])
-	{
-		printf("You must set CGIDIR env var first. Ex. (bash) export CGIDIR=/admin/cgi-bin/ (Note the trailing /)\n");
-		exit(1);
-	}
+	else
+		sprintf(cISMROOT,LIBDIR);
 
 	printf("Creating db and setting permissions, installing data from %s/mysqlBind...\n\n",cISMROOT);
 
@@ -600,9 +574,6 @@
                 }
         }
 
-        sprintf(query,"cp $ISMROOT/mysqlBind/docs/tutorial.html $CGIDIR/mysqlbind.tutorial.txt");
-	system(query);
-
         printf("Done\n");
 	exit(0);
 
@@ -630,9 +601,12 @@
 
         mysql_init(&mysql);
 #ifdef USECONF
-        if(!mysql_real_connect(&mysql,gcEffectiveDbIp,"root",cPasswd,
-					"mysql",guDbPort,gcEffectiveDbSocket,0))
+	if (!mysql_real_connect(&mysql,gcEffectiveDbIp,"mysql",cPasswd,
+				"mysql",guDbPort,gcEffectiveDbSocket,0))
+        if (!mysql_real_connect(&mysql,gcEffectiveDbIp,"root",cPasswd,
+				"mysql",guDbPort,gcEffectiveDbSocket,0))
 #else
+	if (!mysql_real_connect(&mysql,DBIP,"mysql",cPasswd,"mysql",DBPORT,DBSOCKET,0))
         if (!mysql_real_connect(&mysql,DBIP,"root",cPasswd,"mysql",DBPORT,DBSOCKET,0))
 #endif
         {
