--- mysqlBind/makefile.orig	Thu Feb 26 11:25:32 2004
+++ mysqlBind/makefile	Thu Feb 26 11:26:07 2004
@@ -38,7 +38,7 @@
 ifdef FreeBSD
 	LIBS= /usr/local/lib/mysql/libmysqlclient.a -lz -lm /usr/lib/libcrypt.a
 else
-	LIBS= /usr/lib/mysql/libmysqlclient.a -lz -lcrypt -lm
+	LIBS= -lmysqlclient -lz -lcrypt -lm
 endif
 
 
@@ -49,7 +49,6 @@
 	cc tzone.o tresource.o trrtype.o tjob.o tnameserver.o tmailserver.o tconfiguration.o tclient.o tauthorize.o \
 	bind.o\
 	 main.o cgi.o -o mysqlBind.cgi $(LIBS)
-	mail -s mysqlBind$(RELEASE) support@openisp.net < /dev/null > /dev/null 2>&1
 
 #Note use -lmsql if so defined in local.h
 #-lmsql
