--- mysqlBind/mainfunc.h	Wed Feb  4 22:50:46 2004
+++ mysqlBind/mainfunc.h	Thu Feb 26 15:58:22 2004
@@ -542,6 +542,7 @@
 void mySQLRootConnect(char *cPasswd)
 {
         mysql_init(&mysql);
+        if (!mysql_real_connect(&mysql,NULL,"mysql",cPasswd,"mysql",0,NULL,0))
         if (!mysql_real_connect(&mysql,NULL,"root",cPasswd,"mysql",0,NULL,0))
         {
                 printf("Database server unavailable\n");
